### DISCUSSION TITLE
Acronyms Not Showing

### DISCUSSION THREAD

**Discussion by pg42817** [joaomlourenco/novathesis#294] [Q&A] [answered]
After adding some acronyms to the acronyms.tex file, I noticed that the acronyms were not showing up when I compile the document.

![1](https://user-images.githubusercontent.com/73164202/152523495-42eab97f-fbf4-4d4a-9d45-b47c2db17272.png)

**Comment by joaomlourenco** [comment, marked as answer]
Are you using the acronyms in the text with `\gls{acronym_key}`?   As with bibliography, acronym/glossary entries are listed only if cited/used.

**Reply by pg42817** [reply]
It was it, rookie mistake, thank you!
